tests/python-pytest/onnx/onnx_backend_test.py

```diff
@@ -73,12 +73,10 @@
     'test_basic_conv',
     #'test_globalmaxpool',
     #'test_globalaveragepool',
-    #'test_batch_norm',
+    'test_batch_norm',
```

Comment:
onnx backend does not have this tests. See here - https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization
I see BatchNorm has a model test, but could you also please add an operator test.

Reply:
I will create a PR on ONNX repo.

Reply:
will add a test in onnx_test.py